helm/values.yaml

```diff
@@ -15,6 +15,16 @@ storage:
     tag: v0.1.0-alpha1
     pullPolicy: IfNotPresent
   replicas: 3
+  pvc:
+    name: sqlite-pvc
+    requestStorage: 1Gi
+    accessModes:
+      - ReadWriteMany
```

Comment:
`pcv` is cryptic as a name. 

Take inspiration from what other charts do, for example [here](https://github.com/nextcloud/helm/blob/00d8a86ba8bab3f10b7180946dfe37615a071f4c/charts/nextcloud/values.yaml#L597-L613)